### PR TITLE
Create custom 404 and 500 error pages

### PR DESCRIPTION
**Pull request by emilyliublair**
## Create custom 404 and 500 error pages

Issue Number(s): #166 

What does this PR change and why?
Creates custom 404 and 500 error pages

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria is met
- [ ] Database schema [docs](https://www.notion.so/gtbitsofgood/Database-Schema-Docs-fd377ded10b54890b65fd99025488cee?pvs=4) have been updated or are not necessary
- [ ] Relevant reviewers (EM) have been assigned to this PR

### Related PRs

PR Number(s): 

### How To Test

route to /404 and /500 to see error pages

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import errorimage from "public/404-error.svg";
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+export default function Custom404() {
+  const router = useRouter();
+
+  return (
+    <div
+      style={{
```
Comment (afazio1) [inline src/pages/404.js]: Can we use tailwind here for consistency?

**Review by afazio1 — CHANGES_REQUESTED** [review]
Works great just a nit about using tailwind instead of `style`!!

**Review by laurenmcp — APPROVED** [review]
looks great emily! approved provided that alexa's small nit is fixed

**Comment by dograh** [comment]
Yayyy they look great! Awesome job!! Something to keep in mind though is that there are some spacing concerns when adjusting for screen size. When zooming in, the image starts to overlap outside of the white box. Not sure if that's a part of the ticket or not, but just wanted to point that out! Other than that it looks so good!!

**Review by afazio1 — APPROVED** [review]

**Pull request merged**